_solutions/04/E04.04-06.md

```diff
@@ -28,7 +28,7 @@ T(n) & = \sum_{i = 0}^{\log_3 n} cn \\
      & = (\log_3 n + 1) cn \\
      & = cn \log_3 n + cn \\
      & \ge cn \log_3 n \\
-     & = \Omega(2^n)
+     & = \Omega(n\log n)
```

Comment:
Thanks for catching the typo. Could you update the patch to add the extra step to show the exact answer (n lg n), by taking out log_3 out?

Reply:
Yeah sure.

Reply:
Please review.